index.d.ts

```diff
@@ -149,11 +149,11 @@ interface ChalkFunction {
 	(...text: unknown[]): string;
 }
 
-export interface Chalk extends ChalkFunction {
+export interface ChalkInstance extends ChalkFunction {
 	/**
 	Return a new Chalk instance.
 	*/
-	Instance: ChalkInstance;
+	Chalk: ChalkClass;
```

Comment:
Should this still be `Instance` instead of `Chalk`

Reply:
It should be removed here. It's already exported as a named export.